modules/ossm-threescale-applying-external-service-entry-objects.adoc

```diff
@@ -50,23 +67,57 @@ spec:
   resolution: DNS
 ----
 +
-You can use the `oc apply` command with either of the following methods to apply the objects:
+.`DestinationRule` CR
+[source,terminal]
+----
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: destination-rule-threescale-saas-system
+spec:
+  host: multitenant.3scale.net
+  trafficPolicy:
+    tls:
+      mode: SIMPLE
+      sni: multitenant.3scale.net
+----
++
+Use the following commands to apply external `ServiceEntry` and related `DestinationRule` CRs to your cluster:
++
+.3scale Hosted backend and system
+* Save the objects to one or more files, and then use the following syntax:
++
+[source,terminal]
+----
+$ oc apply -f <service-entry-threescale-saas-backend.yml>
+$ oc apply -f <destination-rule-threescale-saas-backend.yml>
+----
+
+* To apply the objects without first saving them to a file, use the following command:
++
+[source,terminal]
+----
+$ echo -n "<service-entry-threescale-saas-backend.yml>" | oc apply -f -
+$ echo -n "<destination-rule-threescale-saas-backend.yml>" | oc apply -f -
+----
 
-** Save the objects to one or more files, and then use the following syntax:
+* Save the objects to one or more files, and then use the following syntax:
 +
 [source,terminal]
 ----
-$ oc apply -f <filename.yml>
+$ oc apply -f <service-entry-threescale-saas-system.yml>
+$ oc apply -f <destination-rule-threescale-saas-system.yml>
 ----
 
-** To apply the objects without first saving them to a file, use the following command:
+* To apply the objects without first saving them to a file, use the following command:
 +
 [source,terminal]
 ----
-$ echo -n "<filename.yml>" | oc apply -f -
+$ echo -n "<service-entry-threescale-saas-system.yml>" | oc apply -f -
+$ echo -n "<destination-rule-threescale-saas-system.yml>" | oc apply -f -
 ----
 
-Alternatively, you can deploy an in-mesh 3scale service. To do this, change the location of these services in the custom resources.
+Alternatively, you can deploy an in-mesh 3scale service. To do this, change the location of these services in the CRs.
```

Comment:
"To do this, change the location of these services in the CRs."  Too many vague pronoun references here, this is going to  confuse anyone who doesn't know the product extremely well.   Name what task you're doing, the services whose location needs to be changed, and which specific CR needs to be updated.